tests/test_security_layer.py

```diff
@@ -2866,6 +2866,58 @@ def test_anonymous_without_password_provider(self):
             SecurityLayer(),
         )
 
+    def test__not_anonymous_without_password_provider(self):
```

Comment:
Change the name to ``test_not_anonymous_without_password_provider`` (note the first underscore).